### PR TITLE
OSDOCS-6154: update networking gateways MicroShift 4.14

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-6154

Link to docs preview:
https://63996--docspreview.netlify.app/microshift/latest/microshift_networking/microshift-networking#microshift-description-connections-network-topology_microshift-networking
https://63996--docspreview.netlify.app/microshift/latest/microshift_networking/microshift-networking#microshift-config-OVN-K_microshift-networking
https://63996--docspreview.netlify.app/microshift/latest/microshift_networking/microshift-networking#microshift-OVS-snapshot_microshift-networking

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@pliurh Please take a look. The image will be updated with a later PR. This is for the text updates only.

**Comment by ShaunaDiaz** [comment]
@dhensel-rh or @jogeo Could someone from QE PTAL?

File: modules/microshift-configuring-ovn.adoc [hunk 1]
```diff
@@ -24,18 +24,14 @@ $ sudo cp /etc/microshift/ovn.yaml.default /etc/microshift/ovn.yaml
...
 ----
-ovsInit:
-  disableOVSInit: false
-  gatewayInterface: "" <1>
 mtu: 1400
 ----
-<1> The default value is an empty string that means "not-specified." The CNI network plugin auto-detects to interface with the default route.
 
-. To customize your configuration, use the following table that lists the valid values you can use:
+. To customize your configuration, you can change the MTU value. The table that follows provides details:
```
Comment (opayne1) [inline modules/microshift-configuring-ovn.adoc:34]: Totally up you you, but just a wording suggestion
```suggestion
. To customize your configuration, the following table provides details for how you can change the MTU value:
```

**Review by opayne1 — COMMENTED** [review]
One suggestion but otherwise lgtm!

File: modules/microshift-configuring-ovn.adoc [hunk 2]
```diff
@@ -47,46 +43,21 @@ mtu: 1400
...
---
-1. The OVS bridge is required. When `disableOVSInit` is true, OVS bridge `br-ex` must be configured manually.
-+
 [IMPORTANT]
 ====
 If you change the `mtu` configuration value in the `ovn.yaml` file, you must restart the host that {product-title} is running on to apply the updated setting.
 ====
---
 
 .Example custom `ovn.yaml` configuration file
```
Comment (mburke5678) [inline modules/microshift-configuring-ovn.adoc:58]: Maybe tuck this example under the note?

**Comment by mburke5678** [comment]
@ShaunaDiaz I had a couple of nits that I am hoping you would consider before merging.

File: modules/microshift-configuring-ovn.adoc (hunk 2 above)
Comment (ShaunaDiaz) [reply modules/microshift-configuring-ovn.adoc:58]: @mburke5678 I don't understand this merge review. You want me to remove the line space? Can you link me to the repo requirement for that, please? I am unfamiliar with the guidance. My understanding is that a line space is needed after the closing of the note? In any case, I hooked these piece together, if that's what you intended?

**Pull request merged**